src/Meshes/transformNode.ts

```diff
@@ -203,7 +203,7 @@ export class TransformNode extends Node {
     /**
      * return true if a pivot has been set
      */
-     public get usePivotMatrix(): boolean {
+     public usePivotMatrix(): boolean {

```

Comment:
Could we change the name to isUsingPivotMatrix or simply add usePivotMatrix to the list of exception in the clone function instead ? :-)